docs/volumes.md

```diff
@@ -3,17 +3,17 @@ This document describes the current state of Volumes in kubernetes.  Familiarity
 
 A Volume is a directory, possibly with some data in it, which is accessible to a Container. Kubernetes Volumes are similar to but not the same as [Docker Volumes](https://docs.docker.com/userguide/dockervolumes/).
 
-A Pod specifies which Volumes its containers need in its [ContainerManifest](https://developers.google.com/compute/docs/containers/container_vms#container_manifest) property.
+A Pod specifies which Volumes its containers need in its [spec.volumes](http://kubernetes.io/third_party/swagger-ui/#!/v1beta3/createPod) property.
```

Comment:
No, it is spec.volumes. The doc also mentions the container's VolumeMounts property in the next paragraph.

Btw, is there any better place we can point people to other than the swagger-ui? swagger-ui currently take too long time to load.